src/pages/LogIn.js

```diff
@@ -47,7 +51,7 @@ const LogIn = () => {
   }
   return (
     <>
-      <h2>Log In</h2>
+      <Heading1 content = {'Log in to your account here'} />
```

Comment:
This is great, we can also drop the {  } when a prop is a string, and I think we want to close up the space here?

```javascript
  <Heading1 content='Log in to your account here' />
```

Reply:
**The spaces between `content` and `=` etc

Reply:
sure!

Reply:
is it a bad practice to use space around = ? I often use it

Reply:
Ah interesting, I think my linter etc always removes it. I am used to always seeing it with no spaces.